README.md

```diff
@@ -38,15 +39,15 @@ Checkout more examples in [demo/index.html](./demo/index.html).
 
 ### `newChild` property
 
-Type: `function(item: any, index: number) => Element`
+Type: `function(itemIndex: number) => Element`
 
-Set this property to configure the virtual list with a factory that creates an element the first time a given item is ready to be displayed in the DOM.
+Set this property to configure the virtual list with a factory that creates an element the first time a given item at the specified index is ready to be displayed in the DOM.
 
 This property is required. Without it set, nothing will render.
 
 ### `updateChild` property
 
-Type: `function(child: Element, item: any, index: number)`
+Type: `function(child: Element, itemIndex: number)`
 
 Set this property to configure the virtual list with a function that will update items' elements.

```

Comment:
This documentation doesn't seem quite updated. This sentence should probably say something like "the list's elements" or "the element at a given index"? And, the second bullet point seems like it should probably just be deleted.

Reply:
Will change to 
> Set this property to configure the virtual list with a function that will update the element **with data** at a given index.

Re second bullet point, we should still mention that `updateChild` is invoked if `size` is changed